src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/model/FindResult.java

```diff
@@ -200,28 +200,32 @@ public final SafeHtml getLineHTML()
             {
                if (parts.remove(0).first)
                {
-                  out.appendHtmlConstant("<strong>");
+                  out.appendHtmlConstant("<span aria-label=\"original\"><strong>");
```

Comment:
I've done some testing with various screen readers, and additional research. This pattern of putting a span inside the `<strong>` and `<em>` does not work as intended and is too awkward for a screen reader user.

The ideal approach is to use the semantically correct HTML: `<del>` element to wrap text being removed (where you currently use `<strong>`) and `<ins>` where you use `<em>`.

These elements reflect what is actually happening, and screen readers are starting to directly support these. For example, NVDA on Firefox speaks "deleted cat", "inserted dog" for `<del>cat</del><ins>dog</ins>`.

VoiceOver on Safari (mac) unfortunately doesn't do this yet. So for coverage there, we'd also want to do the following, note that I'm putting the hidden span INSIDE the `<ins>`, etc. Lots of whitespace added for clarity:

```
<del>
    <span style=visuallyHidden>original</span>
    dog
</del>
<ins>
    <span style=visuallyHidden>replacement</span>
    cat
</ins>
```
To get the `visuallyHidden` style, you have to do it via `ThemeStyles.INSTANCE.visuallyHidden() `.

These elements come with their own styling you'd need to override to meet your needs.

Adding to the complexity, the find results list (when not doing replace) shouldn't use `<del>` or add the embedded `<span>`, it should probably just be `<strong>` on its own.

Not saying you need to do all this now, but I'd definitely NOT do what is in this PR, it actually makes things worse.

Reply:
Based on our slack conversation I added the 'del' and 'ins' tags for replace mode - do you want to take another look?